### ISSUE TITLE
lora extracted from dreambooth trained model has poor compatibility with controlnet

### ISSUE THREAD

**Issue by Nomination-NRB** [kohya-ss/sd-scripts#1892]
Training config([dreambooth_config.json](https://github.com/user-attachments/files/18514231/dreambooth_mahnoor_illustrations_v12_config.json))
Base Model: flux1-dev-fp8-e5m2.safetensors
Steps: 10000
Optimizer: Adafactor
LR Scheduler: constant_with_warmup
Learning rate Unet: 5e-7
Guidance Scale: 1

This is my data:
![Image](https://github.com/user-attachments/assets/723ebbb6-56f0-43e9-8037-8f9c05838a51)

This is my workflow([for_issue_workflow.json](https://github.com/user-attachments/files/18514205/for_issue_workflow.json)):
![Image](https://github.com/user-attachments/assets/f1fd3041-7d80-49ec-9667-6a76b15574ad)
